lib/Containers/SelectWallet.js

```diff
@@ -1,4 +1,4 @@
-import React, { PureComponent } from 'react';
```

Comment:
Just a small nitpick, but this isn't really a SelectWallet container anymore is it? This is basically the entire Wallet interaction view from what I can tell. Something for a potential refactor would be to split these off into stateless components and have this container compose them together.